io/multiwriter.go

```diff
@@ -0,0 +1,46 @@
+package io
+
+import (
+	"errors"
+	"io"
+
+	"golang.org/x/sync/errgroup"
+)
+
+var ErrShortWrite = errors.New("short write")
```

Comment:
What is short write?

Reply:
Changed to
`"The number of bytes written is less than the length of the input"`